guides/release/state-management/computed-properties.md

```diff
@@ -1 +1,634 @@
-Coming Soon!
+Computed properties are a concept in Ember prior to Octane. They serve the same
+purpose as tracked properties and native getters, allowing users to respond to
+changes, derive state, and ultimately update the DOM. They also have built-in
+caching to prevent having to perform expensive calculations more than once.
+
+While computed properties are no longer the recommended default, it's likely
+that you may encounter them in code that hasn't been updated to tracked
+properties just yet, either in existing applications or in the wider Ember
+ecosystem, so this guide exists both to describe how they work and can be used,
+and how they work with tracked properties.
+
+## Computed Property Usage
+
+You can create a computed property by using the `@computed` decorator to
+decorate standard computed property getters and setters:
+
+```javascript
+import { computed, set } from '@ember/object';
+
+class Person {
+  constructor(firstName, lastName) {
+    set(this, 'firstName', firstName);
+    set(this, 'lastName', lastName);
+  }
+
+  @computed('firstName', 'lastName')
+  get fullName() {
+    return `${this.firstName} ${this.lastName}`;
+  }
+}
+
+let ironMan = new Person('Tony', 'Stark');
+
+ironMan.fullName; // "Tony Stark"
+```
+
+This computed property works just like a normal getter/setter, with two key
+differences:
+
+1. It will cache its value by default, and it will only update that value if its
+   _dependencies_, in this case the `firstName` and `lastName` properties,
+   change.
+
+   ```javascript
+   class Counter {
+     _count = 0;
+
+     @computed
+     get count() {
+       console.log('counted!');
+       return this._count;
+     }
+   }
+
+   let counter = new Counter();
+
+   counter.count; // logs 'counted!'
+   counter.count; // logs nothing, the values was cached and hasn't updated
+   ```
+
+2. It will notify other "watchers", such as other computed properties and
+   templates, if any of its dependencies has updated and it needs to be
+   recalculated.
+
```

Comment:
I think maybe we need to add a third point. One of the most significant gotchas of CPs for new devs is that they only run if you are using them in the template. We should state this explicitly somewhere. We can borrow from [explanations in the existing guides](https://guides.emberjs.com/release/object-model/computed-properties/#toc_computed-properties-only-recompute-when-they-are-consumed):

```suggestion
3. A computed property will only recompute its value when it is consumed. Properties are consumed in two ways: by being accessed, like `ironMan.fullName`, or by being referenced in a handlebars template that is currently being rendered, like `{{this.ironMan.fullName}}`.

```